### DISCUSSION TITLE
Extend permissions to only specific editable picklists

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5952] [Feature Requests]
**Is your feature request related to a problem? Please describe.**

> We believe this is a new feature request, however, if there’s a different way to achieve what we are after, please let us know.
> 
> Our collections have four different user roles with progressing permission levels. Currently, only our highest-level user role has the ability to create and modify picklists. This hasn’t been an issue to date as we predominantly have worked with ‘Read-only’ picklists.
> 
> More recently, we started utilising non Read-only (‘open’) picklists in some of our forms to help maintain data quality. For these fields, we would like to give lower level users the ability to add values to a picklist.
> 
> For example, we have created a Visitor Log form that allows Collections to record their visitors, including information where they are from etc. All Collections agreed that even a Guest user, who would have read-only access across the board, should be able to create and modify the Visitor Log entries. As this is a stand-alone table (we are utilising treatmentEvent) that is a pretty straight-forward permission edit.
> 
> However, we have encountered issues with the ‘open’ picklist fields. As the Guest user doesn’t have any editing permissions for the picklists, they cannot add new values to the open picklists. Which sort of defeats the purpose of the setup. The only option to make this functional is to give them full permission to edit ALL picklists. As they cannot edit the schema, currently they cannot get to the picklists via that path. However, if they use a direct link (eg. /specify/view/picklist/31) they have full access to picklists.
> 
> Is there any way we haven’t thought about that would enable us to do what we are after? Alternatively, would it be possible to add a permission option that enables differentiating between Read-only and ‘open’ picklists?

**Describe the solution you'd like**
This would add a new permission option that would enable them to differentiate between 'Read-only' and 'open' picklists for different user roles. This would allow lower level users to add values to 'open' picklists without giving them full permission to edit _all_ picklists.

**Describe alternatives you've considered**
Currently, the only way to make this functional is to give their lower level users full permission to edit all picklists (bad). They have also explored using direct links to access the picklists, but this is not a sustainable solution. 

**Requested by:** CSIRO on [Asana](https://app.asana.com/0/1201972579963460/1207227323056661/f)

**Comment by maxpatiiuk** [comment]
Duplicate of https://github.com/specify/specify7/issues/1957?
